docs/_sources/vc_features.rst.txt

```diff
@@ -0,0 +1,36 @@
+VC features
```

Comment:
Also, I think this might work well in the "editing the meta.yaml" or "building on windows"  No need for a new file.